File: EShipment/client-src/app/model/OrderStatus.ts
export class OrderStatus {
  id: number;
  date: string;
  description: string;
  constructor();
  constructor(id: number, date: string, description: string);
  constructor(id?: number, date?: string, description?: string) {
    this.id = id;
    this.date = date;
    this.description = description;
  }
}

File: EShipment/Models/OrderStatus.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace EShipment.Models
{
    public class OrderStatus
    {
      public OrderStatus()
      {
      }

    public long ID { get; set; }

      [ForeignKey("order")]
      public long Order_Id { get; set; }

      public virtual Order order { get; set; }

      public DateTime? date { get; set; }

      public string description { get; set; }
  }
}
